README.md

```diff
@@ -29,30 +29,104 @@ These public reports help protect the open source community, and provide a data
 source for the security community to improve their ability to find and detect
 new open source malware.
 
-### Scope
+## Scope
 
 What is in scope?
 
 - any package that belongs to an ecosystem supported by the
   [OSV Schema](https://ossf.github.io/osv-schema/)
-- malicious packages published under typosquatting or dependency
-  confusion type attacks
+- malicious packages published under typosquatting type attacks
 - malicious packages published through account takeover
-- prebuilt binaries for a package that are malicious
+- malicious prebuilt binaries downloaded or installed with a package
 - security researcher activity
+- dependency and manifest confusion
+
+Borderline:
+
+- typosquatting, or spam packages that are empty or trivial, while not
+  malicious, are allowed to be present in the dataset
 
 Out-of-scope:
 
+- non-malicious packages
 - vulnerability reports
 - compromised infrastructure
-- non-malicious packages
+- offensive security tools, unless they execute malicious payloads on install
 
-### Prior Work
+## Definition of a Malicious Package
 
-- GitHub's [Advisory Database (filtered by malware)](https://github.com/advisories?query=type%3Amalware), for the NPM ecosystem.
-- https://github.com/lxyeternal/pypi_malregistry (PyPI)
-- https://dasfreak.github.io/Backstabbers-Knife-Collection/ (PyPI and npm), by Marc Ohm et al.
-- https://github.com/datadog/malicious-software-packages-dataset (PyPI), by Datadog
+Below is the definition of what this repository considers a malicious package.
+
+- an open source package publicly available in a package registry
+- and either:
+  - when installed or used, would require some sort of incident response; or
```

Comment:
Minor: definitions are hard, and I wouldn't call myself an expert here, but maybe something like "when installed or runs, tries to persist software on the machine unrelated to the advertised function of the package"